tests/selenium_testing.py

```diff
@@ -0,0 +1,36 @@
+"""
+  Setting up Functional testing for layout web tool.
+"""
+
+import unittest
+import urllib
+
+from flask import Flask
+from flask_testing import LiveServerTestCase
+from selenium import webdriver
+
+class TestBase(LiveServerTestCase):
+
+  def create_app(self):
+      app = Flask(__name__)
+      app.config['TESTING'] = True
+      # Default port is 5000
+      app.config['LIVESERVER_PORT'] = 8943
+      # Default timeout is 5 seconds
+      app.config['LIVESERVER_TIMEOUT'] = 10
+      return app
+
+  def setUp(self):
+    self.driver = webdriver.Chrome()
+
+  def test_Start(self):
+    driver = self.driver
+    driver.get('https://in-toto.engineering.nyu.edu/')
+
+    start = driver.find_element_by_xpath('/html/body/div[3]/div/div/a')
+    start.click()
+
+    self.driver.quit()
+
+if __name__ == '__main__':
+    unittest.main()
```

Comment:
In order for this to work, you need to add these requirements to `requirements.txt`

```
diff --git a/requirements.txt b/requirements.txt
index f23a780..26d84e5 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -2,3 +2,5 @@ Flask==1.1.2
 Flask-PyMongo==2.3.0
 Flask-WTF==0.14.3
 in-toto==0.5.0
+Flask-Testing
+selenium
```